#!/usr/bin/env python2
PACKAGE = "spectral_meter"

from dynamic_reconfigure.parameter_generator_catkin import *
gen = ParameterGenerator()

gen.add("win_size", int_t, 0, "window size", 13, 1, 100)
gen.add("max_level", int_t, 0, "maximum level", 4, 1, 6)
gen.add("min_distance", int_t, 0, "minimum distance between corners", 20, 1, 50)
gen.add("num_features", int_t, 0, "number of features", 500, 1, 1000)
gen.add("min_features_k", double_t, 0, "minimum features", 0.5, 0.0, 1.0)

exit(gen.generate(PACKAGE, "spectral_meter", "OpticalFlowDyn"))
